CONTRIBUTING.md

```diff
@@ -169,14 +169,14 @@ to `.vscode/settings.json` in your local Miri clone:
     ],
     "rust-analyzer.checkOnSave.overrideCommand": [
         "env",
-        "AUTO_OPS=42",
+        "MIRI_AUTO_OPS=no",
```

Comment:
Now that we use this in multiple files we should also document it in the readme with the other internal env vars.